Working udp socket helmsman

thatgirll007 opened the pull request.

vladvan commented:
This branch has source code conflicts and these conflicts need to be resolved before this branch can be merge into master

vladvan commented:
I resolved the conflict if anyone wants to accept the merge request.

vladvan commented:
Might want to remove the branch after accepting the merge request.

Also I will severely injure the next person who puts parentheses in the branch name.

thatgirll007 committed:
Changes with power and distances

changed variables so the shooter could shoot farther or vice versa.
Needs more work

Eric-w-H committed:
resolving conflicts

Eric-w-H committed:
Merge branch 'Working-UDP-Socket-(Helmsman)' of https://github.com/jmchs-robotics/Rosieppbs into Working-UDP-Socket-(Helmsman)
